学生健康辅导中心-Student-Health-and-Counselling-Centre.md

```diff
@@ -0,0 +1,155 @@
+---
+title: 学生健康辅导中心 Student Health and Counselling Centre
+date: 2021-01-29 11:37:17
+tags: 
```

Comment:
Add some tag?

Reply:
Any suggestions? I notice that we don't have tags for Campus Clinic either.
How about `complex`?

Reply:
I mean we can define some tags so in the app we can filter.

Reply:
> I mean we can define some tags so in the app we can filter.

Well, we can leave this issue to tomorrow's discussion (about how to classify).